### PR TITLE
New Serverless Pattern : AWS Lambda (Python) and Amazon Bedrock integration - Terraform

### PR DESCRIPTION
**Pull request by nj80750**
*Issue #, if available:*
https://github.com/aws-samples/serverless-patterns/issues/1978

*Description of changes:*
This pattern will use Amazon Bedrock and AWS Lambda function to show how to develop a GenAI application with minimum code. With this design, customers will be able to choose their preferred Foundation Model from the Amazon Bedrock

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by parikhudit** [comment]
taking a look

**Comment by parikhudit** [comment]
Suggestion: You may want to add a note in README file that the mentioned subnets need to be private and have internet access via NAT Gateway. Alternatively, you may alter the solution to create interface endpoint for bedrock or remove Lambda - VPC association.

File: terraform-lambda-bedrock-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Before attempting this pattern, you must enable [model access](https://docs.aws.amazon.com/bedrock/latest/userguide/model-access.html) in your AWS Account. To provide you flexibility with foundation models, you can enable all or some of the foundation models. This pattern employs the anthropic.claude-v2 model by default so that's must be enabled for this pattern to work
+
+You also need [docker](https://www.docker.com/) and md5 to be installed on your testing machine
+
+## Deployment and Testing Instructions
+
+The deployment will require you to provide the AWS VPC id along with the Subnet id(s) where you want this pattern to be deployed. 
+
+Optionally, you can also specify foundation model name, env and organization (tagging purposes), timeout and memory_size (AWS Lmabda function compute purposes) and lastly logging_level to specify the logging level for your function. These values default to entires in the variable.tf file if not provided at deployment.
```
Comment (parikhudit) [inline terraform-lambda-bedrock-integration/README.md:19]: Typo: 'Lmabda' -> 'Lambda'

**Review by parikhudit — CHANGES_REQUESTED** [review]
Suggestion: You may want to add a note in README file that the mentioned subnets need to be private and have internet access via NAT Gateway. Alternatively, you may alter the solution to create interface endpoint for bedrock or remove Lambda - VPC association.

**Comment by bls20AWS** [comment]
will go live in 24 hours on: https://serverlessland.com/patterns/terraform-lambda-bedrock-integration

**Comment by nj80750** [comment]
Hi @bls20AWS  is it live yet ?

**Pull request merged**